app/views/admin/resource/index.html.erb

```diff
@@ -1 +1,8 @@
-<%= render "index" %>
+<div class="wrap">
+  <%= render partial: "/shared/sidebar" %>
+  <div class="col-xs-8">
+    <%= render "index" %>
+  </div>    
+  <div style="clear:both"></div>       
```

Comment:
also, since we are wrapping and clearing in all the main resource views, this probably suggests placing this in the next higher abstraction. which is probably a custom layout. we could probably abstract what we have from the application layout into partials, and then use a custom layout for the resources/admin side if there is not enough shared wrapper layout.